src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```
@@ -2072,17 +2073,34 @@ public void we_can_collect_the_dropped_messages_stats_from_the_seed_node() throw
             if (Response.Status.OK.getStatusCode() == response.getStatus() && StringUtils.isNotBlank(responseData)) {
               List<DroppedMessages> dropped = SimpleReaperClient.parseDroppedMessagesJSON(responseData);
               if (dropped.isEmpty()) {
-                LOG.error("Got empty response from {}", seed);
+                LOG.error("Got empty DroppedMessages response from {}", seed);
```

Issue: do we want the test to fail out at this point? I think maybe we do.

---

OR, maybe this isn't an error and should be logged as a warning/debug, since in a real environment I guess sometimes messages won't have been dropped...

---

I'm failing fast now